frontend/public/components/utils/hint-block.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+export const HintBlock: React.FC<HintBlockProps> = ({ title, body, className }) => {
+  const classes = classNames('co-hint-block', className);
+
+  return (
+    <div className={classes}>
+      <h2 className="co-hint-block__title h4">{title}</h2>
+      <div className="co-hint-block__body">{body}</div>
+    </div>
+  );
+};
+
+export type HintBlockProps = {
+  title: string;
+  body: React.ReactNode;
```

Comment:
Can we consider making this `children` instead? I know this was the original API from Patternfly, but I don't see a reason this needs to be a special prop.

Reply:
Yeah, we can.  Was going for minimal changes...

Reply:
Updated.  PTAL @spadgett